scipy/integrate/tests/test_quadpack.py

```diff
@@ -538,6 +538,7 @@ def tfunc(x):
 
 
 class TestNQuad:
+    @pytest.mark.filterwarnings('ignore::DeprecationWarning')
```

Comment:
Why not switch straight to the recommended behaviour rather than filtering here? The test added checks the outputs are the same so I think we could just remove `full_output=True`

Reply:
Lazy : )